pkg/mutator/nft_collection.go

```diff
@@ -19,7 +20,7 @@ func (q *NFTCollectionMutator) InsertNFTCollection(blockchain string, network st
 	collection := &eth.NFTCollection{
 		Blockchain:      blockchain,
 		Network:         network,
-		ContractAddress: contractAddress,
+		ContractAddress: strings.ToLower(contractAddress),
```

Comment:
Change GetContractMetadata to take ContractID.
Change InsertNFTCollection to take ContractID.